packages/pci-sdk-iframe/src/pages/app/useApp.ts

```diff
@@ -41,7 +41,7 @@ export default function useApp() {
 
 	useEffect(() => {
 		function _onMessage(event: MessageEvent) {
-			const data = JSON.parse(event.data);
+			const data = event?.data ? JSON.parse(event.data) : {};
```

Comment:
!?!?

this should never happen, why are we doing this check?

Reply:
This went in unintentionally 💆🏽‍♂️ , sometimes when trying to init the SDK locally it'll blow up as `event.data` will come as `undefined` for some reason 🤷🏽‍♂️ hence the check here.